CONTRIBUTING.md

```diff
@@ -27,56 +21,53 @@ All pull requests should be based off of `master`.
 
 ## Propose a single set of related changes
 
-Changes proposed in a single topic branch / pull request should all be related to each other. Don't propose too
-many changes at once, as review becomes challenging. Multiple new features that are loosely coupled should be completed
-in separate topic branches. It is OK if the branch for `feature2` is based on `feature1` - as long as it is made clear
-that `feature1` should be merged before the review of `feature2`. You can merge both `feature1` and `feature2`
-into a temporary integration branch during testing.
+All changes pull request should all be closely related. Multiple change sets that
+are loosely coupled should be proposed in separate pull requests.
 
 ## Keep changes to a minimum
 
-Don't go and "fix" spelling errors all over the code, or make lots of whitespace changes unrelated to the new feature.
+Make changes to files only as necessary.
```

Comment:
Perhaps: "To simplify code review, please avoid making minor changes to code style and documentation in the same pull request as a bug fix or new feature."